packages/firebase_auth/firebase_auth/example/test_driver/firebase_auth_e2e.dart

```diff
@@ -1,3 +1,5 @@
+// @dart = 2.9
```

Comment:
Interesting - could maybe use a link next to the comment itself and a mention that it still needs migration, to connect the idea, I would not have connected "null safety" and a dart version comment without a search, which led to this https://dart.dev/null-safety/unsound-null-safety

assuming I understand it correctly!